### DISCUSSION TITLE
Quarkus & HTMX

### DISCUSSION THREAD

**Discussion by ia3andy** [quarkusio/quarkus#27525] [Community]
I open this thread to discuss about Quarkus & HTMX.

We already have discussed it in different places:
- https://github.com/quarkiverse/quarkus-quinoa/discussions/113
- https://github.com/quarkusio/quarkus/issues/25735

Also here is example app with Quarkus + Renarde + Quinoa and HTMX (made by @tehmoros): 
- https://github.com/TeHMoroS/quarkus-renarde-quinoa-htmx-starter

**Comment by ia3andy** [comment]
cc @maxandersen @FroMage @mkouba @geoand

**Comment by TeHMoroS** [comment]
It's more of a starter than an example app. I've been working on a TodoMVC implementation using the said starter, but had to pause it for a moment. I'll get back on it as soon as possible, so we can have insight on how such a configuration is doing in something more than a starter project. TMVC implementation assumes an example functionality of TodoMVC in a full stack: from the DB (PostgreSQL) to the UI.
